core/test/computed/metrics/largest-contentful-paint-test.js

```diff
@@ -52,6 +52,12 @@ Object {
 
   ['provided', 'simulate'].forEach(throttlingMethod => {
     it(`should fail to compute a value for old trace (${throttlingMethod})`, async () => {
+      // TODO(15841): fails... but instead w/ "No frames found in trace data" (not NO_LCP).
+      // ... handle error when calling code uses lantern? or update expected error? idk
```

Comment:
I'm not sure this test is particularly useful anymore. At this point any pre-LCP trace is going to be complete foobar and we shouldn't need to maintain that `NO_LCP` is always the first error that comes up.